pymongo/mongo_client.py

```diff
@@ -81,7 +81,7 @@ def __init__(
             host=None,
             port=None,
             document_class=dict,
-            tz_aware=False,
+            tz_aware=None,
             connect=True,
```

Comment:
I think `connect` has a similar problem to `tz_aware`, where it can't be specified in the URI. Would you mind making a similar change for connect as you have for tz_aware on L357?